pkg/runtime/containerd/client.go

```diff
@@ -504,10 +505,7 @@ func withMounts(binds []*runtime.Bind) oci.SpecOpts {
 			Source:      bind.HostPath,
 			Destination: bind.ContainerPath,
 			Type:        "bind",
-			Options: []string{
-				"rbind",
-				"rw",
-			},
+			Options: 	append(bind.MountOptions, "rw", "rbind"),
```

Comment:
I suggest:
```
Options:	bind.MountOptions
```
No need to append rw,rwbind here.
This is just a conversion helper for going from ignite's `runtime.Bind{}` to containerd's `specs.Mount{}`.
The options should already be populated in the binds before calling this function.